test/boulder-tools/tag_and_upload.sh

```diff
@@ -3,13 +3,14 @@
 cd $(dirname $0)
 
 DATESTAMP=$(date +%Y-%m-%d)
-BASE_TAG_NAME="letsencrypt/boulder-tools"
-GO_VERSIONS=( "1.15.7" )
+DOCKER_REPO="letsencrypt/boulder-tools"
+
+GO_VERSIONS=( "1.15.7" "1.16" )
```

Comment:
Is there a particular reason you're adding 1.16 here, but not referencing it in the test matrix above? I'd either add it _and_ start testing it, or save both for a later PR.

Reply:
1.16 is currently failing so I took it out of the test matrix. I will remove this and add it in a general 1.16 upgrade PR.